ambari-server/src/main/resources/stacks/HDP/2.3/services/stack_advisor.py

```diff
@@ -337,6 +337,10 @@ def recommendKAFKAConfigurations(self, configurations, clusterData, services, ho
 
     servicesList = [service["StackServices"]["service_name"] for service in services["services"]]
     kafka_broker = getServicesSiteProperties(services, "kafka-broker")
+    kafka_env = getServicesSiteProperties(services, "kafka-env")
+
+    if not kafka_env: #Kafka check not required
```

Comment:
It is unclear how this solves the issue.   Can you comment on that?

Reply:
Seems the UI only sends down configs for related services to the stack_advisor and since Oozie and Kafka are unrelated it leads to this pythonic NPE